pkg/storage/bucket/s3/bucket_client.go

```diff
@@ -1,6 +1,12 @@
 package s3
 
 import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/util/backoff"
```

Comment:
Please group the import. Cortex package should be at the last section separated from other packages we import.